lib/vmdb/console_methods.rb

```diff
@@ -105,5 +105,28 @@ def enable_roles(*role_names)
       MiqServer.my_server.activate_roles(*role_names)
       MiqServer.my_server.active_role_names
     end
+
+    def monitor_vmware
+      require 'VMwareWebService/MiqVim'
+      MiqVim.cacheScope = :cache_scope_core; MiqVim.monitor_updates = true
+    end
```

Comment:
I think @agrare and I had discussed this a long time ago and wanted to see if this could be done in a pluggable way (i.e. have plugins bring their own console methods).  It;s up to Adam if we want to merge as is, or go a little further and have pluggable console methods.

Reply:
Yeah I was trying to check if we were running in a simulate_queue_worker and then do this automatically during vim_connect.  Maybe try that?

Reply:
Originally I was trying to put this in an initializer but simulate_queue_worker is started too late for that to work